stream-tests/src/test/scala/org/apache/pekko/stream/DslFactoriesConsistencySpec.scala

```diff
@@ -41,8 +41,9 @@ class DslFactoriesConsistencySpec extends AnyWordSpec with Matchers {
       "futureFlow", // completionStageFlow
       "futureSink", // completionStageSink
       "lazyFutureSink", // lazyCompletionStageSink
-      "createGraph" // renamed/overload of create for getting type inference working in Scala 3
-    )
+      "createGraph", // renamed/overload of create for getting type inference working in Scala 3
+      // We box Scala Boolean to Java Boolean for better user experience
+      "exists")
```

Comment:
Can this be fixed with a mapping in `scala -> java types`

Reply:
Sure, but it feels like a separate PR just like these works on doc IMO